lightfm/datasets/movielens.py

```
@@ -2,13 +2,204 @@
 import os
 import zipfile
 
-import numpy as np
+# Note: numpy and scipy imports removed for PHP compatibility
+# They are now conditionally imported only when needed in SparseMatrixAdapter
 
 import scipy.sparse as sp
```

Flake8 (7.2.0)

[error] 'scipy.sparse as sp' imported but unused

(F401)